teachable.rb

```diff
@@ -0,0 +1,17 @@
+# Include the following METHODS into this MODULE
+
+  # salary=(new_salary)
+  # receive_raise(raise)
+
+module Teachable
```

Comment:
Teachers are teachable?  I would hope that they are, but I don't think that's quite what you were going for.  Teachers are... capable of teaching.  TeachingCapacity? Not really finding a good name for this.

But, `Teachers` and `ApprenticeTeachers` are both `Instructor`s or `Educator`s, so I'd have them subclass off of `Educator` (which subclasses off `Person`) and avoid this naming issue entirely.